metaflow/cmd/configure_cmd.py

```diff
@@ -7,6 +7,7 @@
 from metaflow.util import to_unicode
 from metaflow._vendor import click
 from metaflow.util import to_unicode
+import metaflow.tracing as tracing
```

Comment:
unused import?

Reply:
Good catch! Would removed in next commit